Amethyst/Managers/FocusTransitionCoordinator.swift

```diff
@@ -159,6 +161,90 @@ class FocusTransitionCoordinator<Target: FocusTransitionTarget> {
         // Otherwise focus the topmost window
         target.executeTransition(.focusWindow(topWindow))
     }
+
+    func moveFocusScreenCounterClockwise() {
+        let focusedWindow = Window.currentlyFocused()
```

Comment:
There isn't always a focused window so this can be `nil`, which will crash on the next line. You need to unwrap it or chain from it.